go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/weaveworks/goformation/v4 v4.10.2-0.20200921152539-943d7e97b454
 	github.com/weaveworks/launcher v0.0.2-0.20200715141516-1ca323f1de15
 	github.com/whilp/git-urls v0.0.0-20191001220047-6db9661140c0
-	golang.org/x/tools v0.0.0-20200502202811-ed308ab3e770
+	golang.org/x/tools v0.0.0-20201013015553-ec925d8b1dc6
```

Comment:
any reason you've updated this? it will require a new docker base image

Reply:
@michaelbeaumont I updated the Go modules while testing locally, I will revert this change if it's a major change.